python/semantic_kernel/connectors/openapi/sk_openapi.py

```diff
@@ -103,11 +104,6 @@ def prepare_request(
                     processed_query_params[param_name] = query_params[param_name]
                 elif param["schema"] and "default" in param["schema"] is not None:
                     processed_query_params[param_name] = param_default
-            elif param["in"] == "header":
```

Comment:
this block should be added back, it allows to instantiate the class with a default set of headers